Libplanet/Tx/BlockPolicyViolationTxException.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace Libplanet.Tx
+{
+    /// <summary>
+    /// The exception that is thrown when a <see cref="Transaction{T}"/> does not follow the
+    /// block policy.
+    /// </summary>
+    [Serializable]
+    public class BlockPolicyViolationTxException : InvalidTxException
```

Comment:
How about `TxViolatingBlockPolicyException`? 🤔

Reply:
Changed to `TxViolatingBlockPolicyException`.